contrib/local/CONTRACTS.md

```diff
@@ -0,0 +1,64 @@
+# Deployment and Instantiation
```

Comment:
can you please move the MD to `docs/resources` dir and call it `TestnetContract.md`